clientd3d/config.c

```diff
@@ -21,18 +21,22 @@
 Config config;
 char inihost[MAXHOST];
 
-// Full pathname of INI file
+// Full pathname of meridian.ini (maintained in-game via the Game>Preferences menu)
```

Comment:
Remove reference to particular menu item.  If the name of that menu item ever changes, no one is going to remember to update this comment.